src/prettytable/prettytable.py

```diff
@@ -2707,6 +2706,7 @@ def _get_formatted_latex_string(self, options: OptionsType) -> str:
 ##############################
 
 
+@functools.lru_cache
```

Comment:
```suggestion
@lru_cache
```

Reply:
Do you think using `@cache` (which has no `maxsize`) would be too much?

Reply:
I don't think its worth it. In very large tables and long running processes you may introduce some kind of potential large "mem leak".